onnx/checker.cc

```diff
@@ -228,6 +228,92 @@ void check_tensor(const TensorProto& tensor, const CheckerContext& ctx) {
 #undef check_field
 }
 
+void check_sequence(const SequenceProto& sequence, const CheckerContext& ctx) {
+  enforce_has_field(sequence, elem_type);
+  if (sequence.elem_type() == SequenceProto::TENSOR) {
+    enforce_has_field(sequence, tensor_values);
+    for (const TensorProto& tensor : sequence.tensor_values()) {
+      check_tensor(tensor, ctx);
+    }
+  }
+  else if (sequence.elem_type() == SequenceProto::SPARSE_TENSOR) {
+    enforce_has_field(sequence, sparse_tensor_values);
```

Comment:
Same here and below for other cases: drop check in repeated case where zero-elements is valid.

Reply:
Thanks for the recommendation. Removed the relevant checks.